### ISSUE TITLE
Improve highlighting on 2020 charter for accessiblity

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/praxis#101]
Highlighting used on the text for the [2020-2021 charter](https://praxis.scholarslab.org/charter/charter-2020-2021/) likely does not pass accessibility requirements for contrast, and should be adjusted to pass.
